src/pudl/metadata/resources/eia860.py

```diff
@@ -154,6 +163,16 @@
                 "generators which may vary from year to year. Compiled from across "
                 "EIA-860 and EIA-923 data."
             ),
+            "additional_details_text": f"""This table contains the attributes about
+EIA-860 generators. Many of these attributes usually do not change year-to-year but
+they occasionally do.
+
+{HARVESTING_DETAIL_TEXT} Most of these generator attributes come from EIA-860 but many
+of them are reported in both EIA-860 and EIA-923 so the values in this table are a
+blend of these two sources.
+
+The related table with the static attributes about generators is
+:ref:`core_eia__entity_generators`.""",
```

Comment:
Do we want a cross-reference like this for all the (`core_eia__entity_X`, `core_eia__scd_X`) pairs?

Reply:
NM I'll cover that in #4704